junifer/external/nilearn/junifer_nifti_spheres_masker.py

```diff
@@ -0,0 +1,255 @@
+"""Provide custom nilearn.maskers.NiftiSpheresMasker tuned for junifer."""
+
+# Authors: Synchon Mandal <s.mandal@fz-juelich.de>
```

Comment:
Is this code from Nilearn? If so, can you adjust the license/info on this file to reflect that?

Reply:
Yeah from nilearn, okay will add the information.

Reply:
Okay two doubts:
- nilearn has a different [style](https://github.com/nilearn/nilearn/blob/c3a9390af2349e3263fd0b2991c17a927a142f6c/nilearn/maskers/nifti_spheres_masker.py#L1) of module docstrings, so it will look way different when it renders.
- According to BSD license (which nilearn uses), needs the [license](https://github.com/nilearn/nilearn/blob/main/LICENSE) to be there in anything that redistributes the code.

Reply:
1) Change the docstrings
2) Keep the header of the file where it says MIT License, including the authors and stuff

Reply:
https://github.com/nilearn/nilearn/blob/98a3ee060b55aa073118885d11cc6a1cecd95059/LICENSE#L4-L35

Reply:
> * Change the docstrings

You mean make it nilearn style?

> * Keep the header of the file where it says MIT License, including the authors and stuff

They have BSD but I get your point.

Reply:
> https://github.com/nilearn/nilearn/blob/98a3ee060b55aa073118885d11cc6a1cecd95059/LICENSE#L4-L35

Shouldn't it be the latest license? Licensing is always tricky.

Reply:
> > * Change the docstrings
> 
> You mean make it nilearn style?

Make them our style




> > https://github.com/nilearn/nilearn/blob/98a3ee060b55aa073118885d11cc6a1cecd95059/LICENSE#L4-L35
> 
> Shouldn't it be the latest license? Licensing is always tricky.

No, the first is nilearn license. The rest are software incorporated in nilearn.

Reply:
I meant that the link you referenced has `2007-2015` and the one I referenced has `2007-2022`. So, should we go for the new one?

Reply:
check the main branch: https://github.com/nilearn/nilearn/blob/main/LICENSE